### DISCUSSION TITLE
New release request

### DISCUSSION THREAD

**Discussion by mikoro** [bottlenoselabs/c2cs#118] [General]
Can a new release be made? Especially interested in the code sorting fix. I could not figure out how to install the latest master version with `dotnet tool install`.

**Comment by lithiumtoast** [comment]
Hey @mikoro,

The release is already made to NuGet, just not reflected to GitHub (it's just a GitHub thing). I also did not update the docs yet on GitHub. See this PR for information on v4 https://github.com/bottlenoselabs/c2cs/pull/108#issue-1362603990. You can see the example of v4 here: https://github.com/bottlenoselabs/c2cs/tree/main/src/cs/examples/helloworld.

To install v4 use `bottlenoselabs.c2cs.tool` instead of `bottlenoselabs.c2cs` for NuGet tool name.

**Reply by mikoro** [reply]
Thanks! Looked into it but the process feels quite a bit more complex compared to the JSON config file. Let's say I want to generate the bindings in CI. Do I need to add a C# project building step to generate the plugin DLL? Adding opaque pre-built DLL to the build process also does not feel right.

**Reply by lithiumtoast** [reply]
I have it setup for CI for some projects already. See https://github.com/bottlenoselabs/sokol-cs/blob/main/.github/workflows/bindgen.yml#L8. You can see how the C# project library plugin is built first before generating the `.json` abstract syntax tree files: https://github.com/bottlenoselabs/sokol-cs/blob/main/bindgen/read_c_code.sh#L4.
